File: Data Structures & Algorithms/sort-an-array/submission-0.cpp
class Solution {
public:
     void bubbleSort(vector<int>& nums,int len){
            int n=0;
            while(n<len-1){
            for(int i=0;i<len-1;i++){
                if(nums[i]>nums[i+1]){
                    int temp=nums[i];
                    nums[i]=nums[i+1];
                    nums[i+1]=temp;
                }
            }
            n++;
            }
           
        }

    vector<int> sortArray(vector<int>& nums) {
        int len=nums.size();

        bubbleSort(nums,len);
        return nums;
    }
};

File: Data Structures & Algorithms/longest-strictly-increasing-or-strictly-decreasing-subarray/submission-0.py
class Solution:
    def longestMonotonicSubarray(self, nums: List[int]) -> int:
        l=0
        m=0
        r=1
        res=1
        ins,des=0,0

        while r<len(nums):
            if nums[m]<nums[r]:
                ins+=1
            else:
                ins=0
                l=m
            
            res=max(res,ins+1)
            m+=1
            r+=1
        k=1
        s=0
        mi=0
        while k<len(nums):
            if nums[mi]>nums[k]:
                des+=1
            else:
                des=0
                s=mi
                # continue
            res=max(res,des+1)
            mi+=1
            k+=1
        return res